README.md

```diff
@@ -48,6 +48,23 @@ For more information, you may [read the docs](https://github.com/intel/e2eAIOK).
 
 ## Installing
 
+### Install with Baremetal Environment
+
+ - To install e2eAIOK in baremetal environment, we recommend using [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/):
```

Comment:
why specific conda?

Reply:
We are referencing [BigDL's implmentation](https://github.com/intel-analytics/BigDL).
Actually users can use conda/virtualenv/pyenv to create seperate virtual env.